tests/enrollment_ca_auto_test.go

```diff
@@ -143,12 +148,35 @@ func Test_IdentityEnrollment(t *testing.T) {
 
 					resp, err := restClient.R().
 						SetHeader("content-type", "application/json").
-						SetBody(`{"name": "` + eid.New() + `"}`).
+						SetBody(`{"name": "` + name + `"}`).
 						Post("enroll?method=ca")
 
 					ctx.Req.NoError(err)
 
 					ctx.Req.Equal(http.StatusOK, resp.StatusCode())
+
+					t.Run("the name was given to the identity", func(t *testing.T) {
+						ctx.testContextChanged(t)
+
+						resp, err := ctx.AdminManagementSession.newAuthenticatedRequest().Get("/identities")
+
+						ctx.Req.NoError(err)
+						standardJsonResponseTests(resp, http.StatusOK, t)
+
+						envelope := rest_model.ListIdentitiesEnvelope{}
+
+						err = envelope.UnmarshalBinary(resp.Body())
+						ctx.Req.NoError(err)
+						found := false
+						for _, identity := range envelope.Data {
+							if *identity.Name == name {
```

Comment:
what happens when the same cased name is provided?

Reply:
This is testing the ability for `[requestedName]`, which is provided during enrollment, is passed through to the controller and applied when required by the name formatting. The value passed in is used opaquely - so whatever the casing is, it is used.